taproot-updates.md

```diff
@@ -137,14 +137,13 @@ The commitment transaction will then become:
             # funds go back to us via a second-stage HTLC-timeout transaction (which contains an absolute delay)
             # NB: we also need the remote signature, which prevents us from unilaterally changing the HTLC-timeout transaction
             <remote_htlcpubkey> OP_CHECKSIGVERIFY <local_htlcpubkey> OP_CHECKSIG
-            1 OP_CHECKSEQUENCEVERIFY
```

Comment:
the remote signature already commits to the sequence of the second stage tx, so explicit `OP_CHECKSEQUENCEVERIFY` is not needed here, saving 2 bytes